cluster/addons/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
```

Comment:
hmm, I discouraged you to make this to cluster/gce/gci...

But I am ok you want to create one specific gci directory under cluster/addons/...